File: ag.consul.030302/tests/jest/authentication.js
function authenticate(targetKey, opts) {
    if (targetKey === "TARGET_CARTS") {
    } else if (targetKey === "TARGET_CATALOGUE") {
    } else if (targetKey === "TARGET_FRONT_END") {
    } else if (targetKey === "TARGET_MOCKINTOSH") {
    } else if (targetKey === "TARGET_ORDERS") {
    } else if (targetKey === "TARGET_USER") {
    } else {
    }
}

module.exports = { authenticate }

File: ag.consul.030302/tests/testify/authentication/authentication.go
package authentication

type Authenticator interface {
    Authenticate(targetKey string)
}

type Authentication struct {

}

type DummyAuth struct {

}

func (a Authentication) Authenticate(targetKey string) {
    switch targetKey {
    case "TARGET_CARTS":
    case "TARGET_CATALOGUE":
    case "TARGET_FRONT_END":
    case "TARGET_MOCKINTOSH":
    case "TARGET_ORDERS":
    case "TARGET_USER":
    }
}

func (a DummyAuth) Authenticate(targetKey string) {
}
